invenio_records_resources/resources/records/resource.py

```diff
@@ -106,6 +108,12 @@ def read(self):
             resource_requestctx.view_args["pid_value"],
             expand=resource_requestctx.args.get("expand", False),
         )
+
+        # emit record view stats event
+        emitter = build_event_emitter("record-view")
+        if item is not None and emitter is not None:
+            emitter(current_app, record=item._record, via_api=True)
```

Comment:
we _could_ emit `record-view` events on `read_draft` (in `invenio-drafts-resources`) as well, but i'm not sure if that's desired?

Reply:
Agree, it wouldn't make sense, since we also don't have a decent place to expose draft statistics. And admins can probably check access logs to investigate any problems.